### ISSUE TITLE
[Accessibility- 10.13]Lose focus on addditionnal content

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3992]
### Problem Statement

Environment: 7.x

#### Steps to Reproduce
1. Add some articles in cards template
2. First Tab into any article to focus the whole article
3. Second Tab to trigger the hover effect for the additional content

#### Current behavior
The focus is lost and the **black border** is no more displayed on the additionnal content

#### Expected behavior

The black border should be displayed when focusing the additionnal content
